ext_emconf.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-$EM_CONF[$_EXTKEY] = [
+$EM_CONF['opencast'] = [
```

Comment:
@franzkugelmann, please revert this change. From https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ExtensionArchitecture/FileStructure/ExtEmconf.html

> Due to limitations of the TER, $_EXTKEY should be used here and not a constant or a string.